sky/engine/core/dart/painting.dart

```diff
@@ -179,8 +180,13 @@ enum VertexMode {
   triangleFan,
 }
 
+/// An interface for recording drawing commands.
```

Comment:
I think we need a better term than "drawing commands", because "drawing" sounds like a verb in all these sentences.
Maybe "graphic commands"?